sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -204,6 +286,44 @@
     "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/vision-classification/valid\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 3. Configure and run the AutoML for Images Classification-Multilabel training job\n",
+    "In this section we will configure and run the AutoML job, for training the model.\n",
+    "\n",
+    "## 3.1 Configure the job through the image_classification_multilabel() factory function\n",
+    "\n",
+    "### image_classification() function parameters:\n",
+    "\n",
+    "The `image_classification_multilabel()` factory function allows user to configure the training job.\n",
+    "\n",
+    "- `compute` - The compute on which the AutoML job will run. In this example we are using a compute called 'gpu-cluster' present in the workspace. You can replace it any other compute in the workspace.\n",
+    "- `experiment_name` - The name of the experiment. An experiment is like a folder with multiple runs in Azure ML Workspace that should be related to the same logical machine learning experiment.\n",
+    "- `name` - The name of the Job/Run. This is an optional property. If not specified, a random name will be generated.\n",
+    "- `target_column_name` - The name of the column to target for predictions. It must always be specified. This parameter is applicable to 'training_data', 'validation_data' and 'test_data'.\n",
+    "- `primary_metric` - The metric that AutoML will optimize for model selection.\n",
+    "- `training_data` - The data to be used for training. It should contain both training feature columns and a target column. Optionally, this data can be split for segregating a validation or test dataset. \n",
+    "You can use a registered MLTable in the workspace using the format '<mltable_name>:<version>' OR you can use a local file or folder as a MLTable. For e.g Input(mltable='my_mltable:1') OR Input(mltable=MLTable(local_path=\"./data\"))\n",
+    "The parameter 'training_data' must always be provided.\n",
+    "\n",
+    "\n",
+    "### set_limits() parameters:\n",
+    "This is an optional configuration method to configure limits parameters such as timeouts.     \n",
```

Comment:
Could you please cleanup set_limits() section in all the image notebooks.

- Only timeout_minutes is applicable to image tasks, others are not supported. Please remove them
- Image tasks don't have setup, featurization, ensemble and model explainability. Please remove the second line in the explanation for timeout_minutes
- I don't think the last line of 10,000,000 rows applies to image tasks. Please remove it as well.

Reply:
Could you also please remove trial_timeout_minutes ? It is not applicable to images.